.gitignore

```diff
@@ -12,3 +12,4 @@ bindata*
 aws_private.pem
 out.json
 .vscode
+static/gen_manifests.go
```

Comment:
I would rather prefer to keep generated code in the repo, but it's absolutelly open for a discussion. @ncopa do you have an opinion about?

Reply:
I actually just cautght a thought that bindata output is non-human readable anyway, so resolving the conversation

Reply:
In my experience, checking in generated code/data into git creates more problems that it solves, and should be avoided.